packages/nhsuk-frontend/src/nhsuk/core/tools/_links.scss

```diff
@@ -125,10 +113,6 @@
   &:active,
   &:active:visited {
     color: $link-active-colour;
-
-    .nhsuk-icon {
-      fill: $link-active-colour;
-    }
```

Comment:
Ah sorry @paulrobertlloyd 

I hated adding these in https://github.com/nhsuk/nhsuk-frontend/commit/cf291cd649246bbb98c2aa198165f3fe4e656b52 but `fill: currentcolor` isn't applied for `:visited` links etc

Action links and cards are hard coded but pagination used these lines 😔

<img width="183" height="110" alt="Pagination arrow, visited state" src="https://github.com/user-attachments/assets/261a9623-aa41-4d54-b853-fd4c5250a432" />

<br>
<br>

The mixins are repetitive but reliable